ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/ProtocolScheduleBuilder.java

```diff
@@ -151,6 +151,19 @@ private ProtocolScheduleBuilder(
             config.getContractSizeLimit(),
             config.getEvmStackSize(),
             isRevertReasonEnabled));
+    addProtocolSpec(
+        protocolSchedule,
+        config.getAtlantisBlockNumber(),
+        MainnetProtocolSpecs.atlantisDefinition(
+            chainId, config.getContractSizeLimit(), config.getEvmStackSize()));
+    addProtocolSpec(
+        protocolSchedule,
+        config.getAghartaBlockNumber(),
+        MainnetProtocolSpecs.aghartaDefinition(
+            chainId,
+            config.getContractSizeLimit(),
+            config.getEvmStackSize(),
+            isRevertReasonEnabled));
```

Comment:
We should update the `validateForkOrdering` method in `ProtocolScheduleBuilder` as well to ensure the ETC forks are applied in the appropriate order.  And we probably should consider it an error if a config is mixing both ETH and ETC forks (except the initial shared ones).

Reply:
I agree, I'll take a look at this.